docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -320,6 +320,53 @@ will be lost.
 $ kubectl delete -f alluxio-journal-volume.yaml
 ```
 
+#### Upgrade Alluxio
+
+Upgrading Alluxio in Kubernetes is easy. You can just change the Alluxio docker image version to your desired upper version.
+Then restart the Pod to make it take effect. You should make sure the Alluxio masters and workers are running on the same version for the best compatibility.
+
+For example, if you are currently running Alluxio v2.0.1 on Alluxio masters and you want to upgrade to the latest version.
+Just change the `image` field of all the Alluxio containers to `image: alluxio/alluxio:latest` and restart them.
+
+```yaml
+  containers:
+  - name: alluxio-master
+    image: alluxio/alluxio:2.0.1
+    imagePullPolicy: IfNotPresent
+    command: ["/entrypoint.sh"]
+    args: ["master-only", "--no-format"]
+    ...
+  - name: alluxio-job-master
+    image: alluxio/alluxio:2.0.1
+    imagePullPolicy: IfNotPresent
+    command: ["/entrypoint.sh"]
+    ...
+```
+
+Whether you need to re-format the journal and lose all file metadata depends on whether the journals are compatible.
+
+* If you are upgrading from v1.8 to v2.x, you have to format the journal. And by doing that you lose all the journal and file metadata.
```

Comment:
Add a bullet **Case A: Embedded Journal**, **Case B: UFS Journal**. keep embedded first as we will recommend that going fwd